### DISCUSSION TITLE
Inserting a large text into a memo gives odd behaviour

### DISCUSSION THREAD

**Discussion by well0549** [Megabit/Blazorise#4470] [Q&A]
I have a modal popup with only three fields. One of them is a memo edit. Now when i insert a large amount of text ionto the memo,
something goes wrong.

Small amounts is ok, but if I insert something of around 14 KB strange things start to happen,

The screen flashes momentarily and my value (that i just inserted) is empty.....
The text is shown in the MemoEdit, but is no longer bound to the string value.

**Comment by stsrki** [comment]
Can you share your code?

**Reply by well0549** [reply]
Found wat it was.......
Nothing to do with this Great product.

But with Blazor Server.....
after adding:

    builder.Services.AddServerSideBlazor()
    .AddHubOptions(options => { options.MaximumReceiveMessageSize = 64 * 1024; });

I could entre the text with no problems.......

**Reply by well0549** [reply]
But that was a quick response 👍  great to see it is actively supported !!!!!!!!!!

**Reply by stsrki** [reply]
Thanks for sharing the solution.

> But that was a quick response 👍 great to see it is actively supported !!!!!!!!!!

Of course, it is. We're now stopping 💪
